nimoy/assertions/mocks.py

```diff
@@ -0,0 +1,17 @@
+class MockAssertions:
+    def assert_mock(self, number_of_invocations, mock, method, *args):
```

Comment:
You might need to support `**kwargs` too. They're quite popular and are supported by mocks.